From: David Heidelberg <david@ixit.cz>
Date: Tue, 16 Sep 2025 12:16:01 +0200
Subject: fixup! power: supply: qcom_smbx: add smb5 support

invalid syntax, must be used with [], not reference

Breaks compilation.

Signed-off-by: David Heidelberg <david@ixit.cz>
---
 drivers/power/supply/qcom_smbx.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/power/supply/qcom_smbx.c b/drivers/power/supply/qcom_smbx.c
index c10e5da..2b5d20b 100644
--- a/drivers/power/supply/qcom_smbx.c
+++ b/drivers/power/supply/qcom_smbx.c
@@ -254,7 +254,7 @@ struct smb_match_data {
 	const char *name;
 	enum smb_generation gen;
 	size_t init_seq_len;
-	const struct smb_init_register __counted_by(init_seq_len) *init_seq;
+	const struct smb_init_register *init_seq;
 };
 
 static enum power_supply_property smb_properties[] = {
